charts/kaito/workspace/templates/clusterrole.yaml

```diff
@@ -21,6 +21,9 @@ rules:
   - apiGroups: [ "" ]
     resources: [ "pods"]
     verbs: ["get","list","watch","create", "update", "patch" ]
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs: [ "get","list","watch","create", "delete" ]
```

Comment:
Do we really need to "create" or "delete" configmaps in the kaito controller?

Reply:
no removed